front/app/containers/AggDropDown/queries/SearchPageAggBucketsQuery.ts

```diff
@@ -0,0 +1,37 @@
+import gql from 'graphql-tag';
```

Comment:
I tried using graphql-tag's webpack loader and it did not want to cooperate with TypeScript. Real bummer, because you lose a lot of IDE functionality by not being able to treat a file like this as a `.gql`.

Reply:
It is a bit of a bummer. Usually though, I write my graphql queries in graphiql so losing the ide features here is less painful.